NotificationApi/NotificationApi.Contract/Responses/NotificationTemplateResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace NotificationApi.Contract.Responses
+{
+    public class NotificationTemplateResponse
+    {
+        public long Id { get; set; }
+
+        public Guid NotifyemplateId { get; set; }
+
+        public int NotificationType { get; set; }
```

Comment:
Although there are only 2 notification types at present, the list is expected to grow and we don't need the client to have to know all possible values

Reply:
done :)